### PR TITLE
Create developers.md

### PR DESCRIPTION
**Pull request by PeishanFromMixpanel**
moving overview section of mixpanel for developers into the landing page

### CODE REVIEW

**Comment by PeishanFromMixpanel** [comment]
I created a landing page for how to use mixpanel, and so want to remove the [overview page](https://docs.mixpanel.com/docs/other-bits/tutorials/mixpanel-analysis/overview). do i simply delete the file and the meta.json link, or do i need to create a redirect also?

File: pages/docs/other-bits/tutorials/mixpanel-analysis.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Overview of tutorial: How to use Mixpanel
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/mixpanel-analysis.md]: capitalization

File: pages/docs/other-bits/tutorials/mixpanel-analysis.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+In this tutorial, we will explore the fundamentals of how Mixpanel works and highlight some of the key features to get you started. 
+
+Log in to your Mixpanel account to follow along. Don't have an account yet? Get your free Mixpanel demo account [here](https://mixpanel.com/project/3/view/79/app/boards) to access the demo datasets.
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/mixpanel-analysis.md]: this link shouldn't be to project 3 right? maybe link directly to a demo project?

File: pages/docs/other-bits/tutorials/how-to-clean-your-data.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### Cleaning up display names and adding descriptions for events
+
+It’s always helpful to add descriptions to your events so users know where they are triggered. If you’re an owner or admin, you can add the descriptions directly on the UI itself. You could also export the data into a CSV, update the description there, then import it back into Lexicon if it’s easier to clean up that way. If you leverage Segment, mParticle, or Avo.app, you can use that to import your event names and descriptions that way too.
+
+If you use Figma to identify your events, some customers add their links to the event descriptions as well.
+
+### Adding tags to events
+
+<p style={{ position: 'relative', paddingBottom: '56.25%', height: 0, overflow: 'hidden' }}>
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/how-to-clean-your-data.md]: put these in divs

File: pages/docs/other-bits/tutorials/how-to-clean-your-data.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  <iframe src="https://www.loom.com/embed/caa348ddf65a44a7b7c24adfd928cb29" frameborder="0" style={{ position: 'absolute', top: 0, left: 0, width: '100%', height: '100%' }} allowfullscreen></iframe>
+</p>
+
+## Using custom events to combine or filter events
+<p style={{ position: 'relative', paddingBottom: '56.25%', height: 0, overflow: 'hidden' }}>
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/how-to-clean-your-data.md]: same

File: pages/docs/other-bits/tutorials/how-to-clean-your-data.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+## Cleaning up data when it gets messy
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/how-to-clean-your-data.md:1]: this file isn't linked in your tutorial overview page

File: pages/docs/other-bits/tutorials.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+## Mixpanel Tutorials
+We've collated here educational resources to guide you through best practices on what and how to implement on Mixpanel, and how to leverage Mixpanel to answer your analytics questions. 
+
+- [Setting Up Mixpanel](https://docs.mixpanel.com/docs/other-bits/tutorials/setting-up-mixpanel): Best practice recommendations on how to set up and organise your Mixpanel organisation and projects.
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials.md]: all of these links should use relative paths; basically remove the "https://docs.mixpanel.com" prefix on all of these links

**Review by mavlee — COMMENTED** [review]
generally these changes look structurally good

https://docs-almqqtfrs-mixpanel.vercel.app/docs/other-bits/tutorials/how-to-clean-your-data

you can see the preview by looking at the vercel comments

**Pull request merged**